.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
> Why using source avoid caching the vendor folder?

Don't really now. But I used this method for my project before and this is not used at all.

> Also if you print the ENV variables you will get the token available, so anyone can get it.

False! Just see this Travis job of my project: https://travis-ci.org/Soullivaneuh/IsoCodesValidator/jobs/65915507#L83 ;-)